artifactory/commands/transferfiles/utils.go

```diff
@@ -333,19 +333,13 @@ func reduceCurProcessedChunks() {
 	curProcessedUploadChunks--
 }
 
-func handleFilesOfCompletedChunk(chunkFiles []FileUploadStatusResponse, errorsChannelMng *ErrorsChannelMng, phase *phaseBase) (stopped bool) {
+func handleFilesOfCompletedChunk(chunkFiles []FileUploadStatusResponse, errorsChannelMng *ErrorsChannelMng) (stopped bool) {
 	for _, file := range chunkFiles {
 		switch file.Status {
 		case Success:
-			if phase != nil && phase.phaseId == ErrorsPhase {
-				phase.progressBar.changeNumberOfFailuresBy(-1)
-			}
 		case SkippedMetadataFile:
 			// Skipping metadata on purpose - no need to write error.
```

Comment:
Can we leave only the last case and put the rest as a 'Default' case? Did you keep it for readability?

Reply:
You're right. To simplify the code, I replaced the `switch` with a an `if` statement.

Reply:
This simplification has been implemented in https://github.com/jfrog/jfrog-cli-core/pull/567